l1-contracts-foundry/script/RegisterHyperchain.s.sol

```diff
@@ -76,14 +51,27 @@ contract RegisterHyperchainScript is Script {
         registerHyperchain();
         addValidators();
         configureZkSyncStateTransition();
+        setPendingAdmin();
 
         saveOutput();
     }
 
+    // This function should be called by the owner to accept the admin role
+    function acceptAdmin() public {
```

Comment:
typically `admin` will be [governance.sol](https://github.com/matter-labs/era-contracts/blob/main/l1-contracts/contracts/governance/IGovernance.sol) contract (this is how we plan to track for upgrades), better to have a way encode a call to this contract, instead of EOA accepting admin